seacatauth/authz/role/view/global_propagated_role.py

```diff
@@ -0,0 +1,85 @@
+import re
+import typing
+
+from .abc import RoleView
+
+
+class GlobalPropagatedRoleView(RoleView):
```

Comment:
I suggest to use "propagated" name only.

Reply:
good point, that was misleading.